manifests/params.pp

```diff
@@ -14,6 +14,7 @@
   $plugin_version = 'installed'
 
   # variables
+  $bind_host = '*'
```

Comment:
In fact, I think we deliberately chose '*' as the default for that reason.. 0.0.0.0 was proposed originally.